src/Paket.Core/PaketConfigFiles/InstallModel.fs

```diff
@@ -780,12 +802,7 @@ type InstallModel with
     [<Obsolete("usually this should not be used")>]
     member this.GetReferenceFolders() = InstallModel.getCompileLibFolders this
 
-    //member this.MapFolders mapfn = InstallModel.mapFolders mapfn this
-
-    //member this.MapFiles mapfn = InstallModel.mapFiles mapfn this
 
-    [<Obsolete("usually this should not be used, use GetLegacyReferences for the full .net and GetCompileReferences/GetRuntimeLibraries for dotnetcore")>]
-    member this.GetLibReferences target = InstallModel.getLegacyReferences target this
```

Comment:
It's been a full major version, I'm sick of the warnings filling up the entire build logs. If it its obsolete it needs to be upgraded at some point.